### PR TITLE
docs(aiplatform): Change all samples to use inline initialization #2626

### PR DESCRIPTION
**Pull request by meteatamel**
Updated all samples to initialize requests inline per @jskeet's suggestion

### CODE REVIEW

File: aiplatform/api/AIPlatform.Samples/MultimodalMultiImage.cs [hunk 1]
```diff
@@ -49,60 +46,30 @@ public class MultimodalMultiImage
...
             }
-        });
-
-        var generateContentRequest = new GenerateContentRequest
-        {
-            Model = $"projects/{projectId}/locations/{location}/publishers/{publisher}/models/{model}"
         };
-        generateContentRequest.Contents.Add(content);
 
-        // Make the request, returning a streaming response
```
Comment (jskeet) [inline aiplatform/api/AIPlatform.Samples/MultimodalMultiImage.cs:100]: Were the comments here and at line 105 deliberately removed?

Comment (jskeet) [reply aiplatform/api/AIPlatform.Samples/MultimodalMultiImage.cs:100]: (Ditto other samples)

**Review by jskeet — APPROVED** [review]
Not sure it was worth removing the comments, but I'm fine with it if that was deliberate.

Personally I find the new style *much* easier to understand in terms of the structure of the request - thanks!

File: aiplatform/api/AIPlatform.Samples/MultimodalMultiImage.cs (hunk 1 above)
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/MultimodalMultiImage.cs:100]: Yes. I only kept comments in the GeminiQuickstart but the rest wasn't really adding much value.

**Review by amanda-tarafa — APPROVED** [review]
Oh yes, nice!

**Pull request merged**